mne/epochs.py

```diff
@@ -241,19 +241,7 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
             raise ValueError('detrend must be None, 0, or 1')
 
         # check that baseline is in available data
-        if baseline is not None:
-            baseline_tmin, baseline_tmax = baseline
-            tstep = 1. / info['sfreq']
-            if baseline_tmin is not None:
-                if baseline_tmin < tmin - tstep:
-                    err = ("Baseline interval (tmin = %s) is outside of epoch "
-                           "data (tmin = %s)" % (baseline_tmin, tmin))
-                    raise ValueError(err)
-            if baseline_tmax is not None:
-                if baseline_tmax > tmax + tstep:
-                    err = ("Baseline interval (tmax = %s) is outside of epoch "
-                           "data (tmax = %s)" % (baseline_tmax, tmax))
-                    raise ValueError(err)
+        _check_baseline(baseline, tmin, tmax, info['sfreq'])
         if tmin > tmax:
             raise ValueError('tmin has to be less than or equal to tmax')
```

Comment:
Maybe this should go above the ``check_baseline``. Otherwise the error message would usually be about baseline being off.